src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -711,7 +711,7 @@ private function registerProfilerConfiguration(array $config, ContainerBuilder $
         }
 
         $container->setParameter('profiler_listener.only_exceptions', $config['only_exceptions']);
-        $container->setParameter('profiler_listener.only_master_requests', $config['only_master_requests']);
+        $container->setParameter('profiler_listener.only_main_requests', $config['only_main_requests'] || $config['only_master_requests']);
```

Comment:
What's our policy on such parameters? I assumed that we only use that parameter internally, so it is safe to rename it. Alternatively, I could set the old parameter as an alias to the new one.

Reply:
Let's consider it internal I agree. Let's just mention it in the changelog/upgrade files?

Reply:
Done.